[2023-08-16T05:50:29.700+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag.inaccurate manual__2023-08-16T05:50:24.849063+00:00 [queued]>
[2023-08-16T05:50:29.706+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag.inaccurate manual__2023-08-16T05:50:24.849063+00:00 [queued]>
[2023-08-16T05:50:29.706+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-16T05:50:29.713+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): inaccurate> on 2023-08-16 05:50:24.849063+00:00
[2023-08-16T05:50:29.717+0000] {standard_task_runner.py:57} INFO - Started process 100 to run task
[2023-08-16T05:50:29.720+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'manual__2023-08-16T05:50:24.849063+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmprf9r6mhk']
[2023-08-16T05:50:29.722+0000] {standard_task_runner.py:85} INFO - Job 14: Subtask inaccurate
[2023-08-16T05:50:29.753+0000] {task_command.py:410} INFO - Running <TaskInstance: my_dag.inaccurate manual__2023-08-16T05:50:24.849063+00:00 [running]> on host 55f7ca6c9533
[2023-08-16T05:50:29.805+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2023-08-16T05:50:24.849063+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-16T05:50:24.849063+00:00'
[2023-08-16T05:50:29.807+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-16T05:50:29.807+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "echo 'inaccurate'"]
[2023-08-16T05:50:29.815+0000] {subprocess.py:86} INFO - Output:
[2023-08-16T05:50:29.818+0000] {subprocess.py:93} INFO - inaccurate
[2023-08-16T05:50:29.818+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-16T05:50:29.838+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20230816T055024, start_date=20230816T055029, end_date=20230816T055029
[2023-08-16T05:50:29.851+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-16T05:50:29.864+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
